alert-policies/redis-prometheus/client-connections-alert.yml

```
@@ -0,0 +1,26 @@
+name: client connections alert
```

Let's make this sentence case since the other alert is!
```suggestion
name: Client connections alert
```